packages/foreman/rubygem-ruby2_keywords/rubygem-ruby2_keywords.spec

```diff
@@ -0,0 +1,83 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name ruby2_keywords
+%global gem_require_name %{gem_name}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.4
+Release: 1%{?dist}
+Summary: Shim library for Module#ruby2_keywords
```

Comment:
I have a slight hunch that this won't render nicely ;)

Reply:
Updated